CHANGES.md

```diff
@@ -1,6 +1,8 @@
 ## VERSION X.X
 
-(TBD)
+* \#206: Upgraded to JTS 1.18.1.  This JTS release had an incompatible [API change](https://github.com/locationtech/jts/blob/master/doc/JTS_Version_History.md#api-changes) and it requires Java 1.8.
+  Spatial4J will not work anymore with older versions.
```

Comment:
```suggestion
  Only Spatial4j's `JtsBinaryCodec` was impacted, and thus only use of this will require a JTS upgrade.
```

Reply:
Hmm; it seems I can't apply my suggested change.  Assuming you agree with it, can you apply it?  Then I'll squash-merge the PR.

Reply:
Yeap. works for me.